### PR TITLE
Adding amm methods

### PR DESCRIPTION
**Pull request by tinystarinagalaxy**
stream examples working 

streaming pool reserves...
{"slot": "166868966", "reserves": {"token1Reserves": "504197195494", "token1Address": "7vfCXTUXx5WJV5JADk17DUJ4ksgau7utNKj4b963voxs", "token2Reserves": "504197195494", "token2Address": "EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v", "poolAddress": "EoNrn8iUhwgJySD1pHu8Qxm5gSQqLK3za4m8xzD2RuEb", "project": "P_RAYDIUM"}}
streaming price streams...
{"slot": "166869020", "price": {"token": "wSOL", "tokenAddress": "So11111111111111111111111111111111111111112", "project": "P_RAYDIUM", "buy": 14.68935, "buySize": 1, "sell": 14.615878, "sellSize": 1}}

### CODE REVIEW

File: example/provider/main.py [hunk 1]
```diff
@@ -98,6 +98,29 @@ async def grpc():
...
+    step.out_amount = "0.01"
+    step.out_amount_min = "0.01"
+    stepProject = proto.StepProject()
+    stepProject.label = "Raydium"
+    stepProject.id = "1234"
+    step.project = stepProject
+    print((await api.post_route_trade_swap(project=proto.Project.P_RAYDIUM,
+                                           owner_address=PUBLIC_KEY, steps=[step])).to_json())
+
+
```
Comment (aspin) [inline example/provider/main.py:128]: example for pool reserves stream?

Comment (aspin) [reply example/provider/main.py:128]: also, I think prices endpoint and prices stream? (though those are "universal")

Comment (tinystarinagalaxy) [reply example/provider/main.py:128]: done

Comment (aspin) [reply example/provider/main.py:128]: still need get price endpoint

Comment (tinystarinagalaxy) [reply example/provider/main.py:128]: added

**Review by aspin — APPROVED** [review]

**Pull request merged**